protoc-gen-kotlin/lib/src/commonMain/kotlin/pbandk/gen/CodeGenerator.kt

```diff
@@ -121,37 +166,114 @@ open class CodeGenerator(
         // doesn't allow cyclic frozen structures:
```

Comment:
Move this comment block to be back above the line with the `by lazy` declaration